src/zac/accounts/admin.py

```diff
@@ -17,7 +19,8 @@ class UserAuthorizationProfileInline(admin.TabularInline):
 
 
 @admin.register(User)
-class _UserAdmin(UserAdmin):
+class _UserAdmin(UserAdmin, HijackUserAdminMixin):
```

Comment:
```suggestion
class _UserAdmin(HijackUserAdminMixin, UserAdmin):
```
mixins should always go before the base class(es) because of [MRO](https://www.python.org/download/releases/2.3/mro/)